src/engines/synths/goss/goss.hpp

```diff
@@ -8,14 +8,34 @@
 #include <Gamma/Filter.h>
 #include <Gamma/Oscillator.h>
 
+#include "util/reflection.hpp"
+
 namespace otto::engines {
 
   using namespace core;
   using namespace core::engine;
   using namespace props;
 
-  struct GossSynth : SynthEngine, EngineWithEnvelope {
+  template<typename Engine>
+  struct AudioData;
+
+  struct Goss {};
+
+  template<>
+  struct AudioData<Goss> {
+    float drawbar1 = 1;
+    float drawbar2 = 0.5;
+    float drawbar3 = 0.5;
+    float leslie = 0.3;
+
+    float rotation_value = 0;
+
+    DECL_REFLECTION(AudioData<Goss>, drawbar1, drawbar2, drawbar3, ("leslie_value", leslie))
```

Comment:
An example of how reflection is declared for a struct. the `leslie_value` is to show that we can also set separate names for values